chainntnfs/txconfnotifier.go

```diff
@@ -297,6 +322,42 @@ func (tcn *TxConfNotifier) ConnectTip(blockHash *chainhash.Hash,
 		}
 	}
 
+	// In order to update the height hint for all the required transactions
+	// under one DB transaction, we'll gather the set of unconfirmed
+	// transactions along with the ones that confirmed at the current
+	// height.
+	var txs []chainhash.Hash
```

Comment:
equivalent to `txs = tcn.txsByInitialHeight[tcn.currentHeight]`?

Reply:
Naming suggestion: `txnsToUpdateHints`, or something along those lines.

Reply:
Renamed to `txsToUpdateHints`.